src/v/kafka/server/group.cc

```diff
@@ -103,6 +109,10 @@ group::group(
         vlog(_ctxlog.trace, "Initializing group with member {}", member);
         add_member_no_join(member);
     }
+
+    if (_enable_group_metrics) {
```

Comment:
I guess it's debatable whether we want this check for the new endpoint.

Reply:
Let's take this check out. I see this check as redundant since we have a config parameter for this.

Reply:
done.